library/core/src/main/java/com/google/android/exoplayer2/Format.java

```diff
@@ -167,6 +167,16 @@
    */
   public final int accessibilityChannel;
 
+  /**
+   * The Role descriptor value.
+   */
+  public final String role;
```

Comment:
Note that video formats can have role/accessibility as well. As can selection flags actually. So these three params shouldn't be under "// Audio and text specific.". They should probably be right up under the `label` param, or somewhere near there.

Reply:
Then I guess the parsing also needs to be moved up, right now it is called only in case of `mimeTypeIsRawText(sampleMimeType) == true` in `DashManifestParser`. Right?